### ISSUE TITLE
Dropbox has blurry borders in Chromium browsers

### ISSUE THREAD

**Issue by brandlfp** [sa-si-dev/virtual-select#299]
The dropbox `<div class="vscomp-dropbox">` has blurry borders if the dropbox container `<div id="vscomp-dropbox-container-1034" role="listbox" class="vscomp-dropbox-container pop-comp-wrapper position-bottom" data-from-left="774.59375" data-from-top="229.25" data-top="259.25" data-left="774.59375" style="z-index: 251; display: inline-flex; transform: translate3d(774.594px, 259.25px, 0px); max-width: 400px; transition-duration: 0ms; opacity: 1;">` has `transform: translate3d` with subpixel (decimal) values applied. This is due to a known issue in Chromium browsers.

![image](https://github.com/sa-si-dev/virtual-select/assets/156655417/78d4b5d7-e207-4c90-a7e6-99dcd4319cfb)

The problem can be fixed by rounding the coordinates to the nearest integer: `<div id="vscomp-dropbox-container-1034" role="listbox" class="vscomp-dropbox-container pop-comp-wrapper position-bottom" data-from-left="774.59375" data-from-top="229.25" data-top="259.25" data-left="774.59375" style="z-index: 251;display: inline-flex;transform: translate3d(775px, 259px, 0px);max-width: 400px;transition-duration: 0ms;opacity: 1;">`

![image](https://github.com/sa-si-dev/virtual-select/assets/156655417/25b6d12f-93e5-4db8-ab1a-03aa8f95a2ba)

**Comment by gnbm** [comment]
@brandlfp I did some research and the issue is actually on a dependency that is included in the generated code which is the [popover](https://github.com/sa-si-dev/popover/) that depends on [popper](https://github.com/sa-si-dev/popper/) where the problem is.
I tried to fix this on this [PR](https://github.com/sa-si-dev/popper/pull/1) on popper but I'm struggling to have any feedback from @sa-si-dev since he's the owner and I have no permission to merge or create releases on this repo and the same on popover that would need to get its dependency updated to this new fixed version so, we could finally update it in Virtual select.
Any help from @sa-si-dev or anyone who knows him or has access to these 3 repos is highly appreciated.
